normits_demand/utils/pandas_utils.py

```diff
@@ -13,10 +13,11 @@
 # Builtins
```

Comment:
Tagging this file to watch out for. 

I need to get moving on #962 which aims to remove the pandas_utils functions over to caf.toolkit